ingestion/src/metadata/utils/dbt_config.py

```diff
@@ -143,7 +174,14 @@ def _(config: DbtS3Config):
                 if DBT_CATALOG_FILE_NAME in bucket_object.key:
                     logger.debug(f"{DBT_CATALOG_FILE_NAME} found")
                     dbt_catalog = bucket_object.get()["Body"].read().decode()
-        return json.loads(dbt_catalog), json.loads(dbt_manifest)
+                if DBT_RUN_RESULTS_FILE_NAME in bucket_object.key:
+                    logger.debug(f"{DBT_RUN_RESULTS_FILE_NAME} found")
+                    dbt_run_results = bucket_object.get()["Body"].read().decode()
+        return (
+            json.loads(dbt_catalog),
+            json.loads(dbt_manifest),
+            json.loads(dbt_run_results),
```

Comment:
have we tested this if the user does not send the Run Results?

Reply:
Yes tested this it works as normal DBT without tests if run_results.json file is not provided